README.md

```diff
@@ -69,6 +69,7 @@ To change log level export OPENSHIFT_PYTHON_WRAPPER_LOG_LEVEL:
 ```bash
 export OPENSHIFT_PYTHON_WRAPPER_LOG_LEVEL=<LOG_LEVEL> # can be: "DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"
 ```
+Important: DEBUG log level should be used with caution. It prints unhashed raw data for resources in the logs, which may expose sensitive information. Use only in secure environments.log.
```

Comment:
```suggestion
Important: DEBUG log level should be used with caution. It prints unhashed raw data for resources in the logs, which may expose sensitive information. Use only in secure environments.
```

I think that only the readme is not enough; because we want to make sure no sensitive data is logged, we should have a dedicated env var for this.

Reply:
this comment was unresolved

Reply:
I think it's too much, user needs to enable debug via env var, user should read the README.

Reply:
they should but exposing confidential info because it was missed is too risky